### PR TITLE
Add Action Network

### PR DESCRIPTION
**Pull request by j-ro**

### CODE REVIEW

File: _data/communication.yml [hunk 1]
```diff
@@ -1,4 +1,11 @@
 websites:
+    - name: Action Network
+      url: https://actionnetwork.org/
+      img: actionnetwork.png
+      tfa: Yes
+      software: Yes
+      doc: https://help.actionnetwork.org/hc/en-us/articles/217206826
```
Comment (Jawshy) [inline _data/communication.yml:7]: The site does not appear to have any public documentation regarding their implementation of 2-factor authentication. The `doc` link only prompts the user to sign in to "[enroll in 2-factor authentication](https://help.actionnetwork.org/hc/en-us/articles/203112329-Edit-your-profile-avatar-email-address-name-password-or-homepage)..." 

@j-ro, would you be able to attach a screenshot that shows how the site has implemented software 2FA?

**Review by Jawshy — CHANGES_REQUESTED** [review]
Regarding `doc`.

File: _data/communication.yml (hunk 1 above)
Comment (j-ro) [reply _data/communication.yml:7]: Sure, yes. We use Instant2FA for our authentication service. (https://instant2fa.com/)

By attach a screenshot, you mean here in this PR? Or in our help documentation?

Comment (Jawshy) [reply _data/communication.yml:7]: In this PR, please.

Comment (j-ro) [reply _data/communication.yml:7]: Sure. After you create an account you can edit your profile and enroll in 2FA: 

https://www.evernote.com/l/AD_LlFRUKd1D9qT4EwzyKP-bCbcmrpCodPk

From then on, when you log in, you will be prompted for your 2-factor token:

https://www.evernote.com/l/AD_tGk5KcPxDfZ0RljWNWeooxW4xdE8SOWE

You can edit your 2-factor settings from your profile page:

https://www.evernote.com/l/AD8j0Sk0YmZC5IAW9F7O1Zp5_sAxH7HVa2U

Comment (Jawshy) [reply _data/communication.yml:7]: Thanks, @j-ro.

**Review by Jawshy — APPROVED** [review]

**Comment by Carlgo11** [comment]
Thanks for the PR @j-ro!
Your commits will go live once I've manually merged to the site. It will probably happen tomorrow.

**Comment by j-ro** [comment]
Thanks @Carlgo11, appreciate it!

**Pull request merged**